base/src/main/java/com/smartdevicelink/managers/lifecycle/BaseSystemCapabilityManager.java

```diff
@@ -266,81 +272,90 @@ void parseRAIResponse(RegisterAppInterfaceResponse response) {
+                                if (onSystemCapabilityUpdated.getSystemCapability() != null) {
+                                    SystemCapability systemCapability = onSystemCapabilityUpdated.getSystemCapability();
+                                    SystemCapabilityType systemCapabilityType = systemCapability.getSystemCapabilityType();
+                                    Object capability = systemCapability.getCapabilityForType(systemCapabilityType);
+                                    if (cachedSystemCapabilities.containsKey(systemCapabilityType)) { //The capability already exists
+                                        switch (systemCapabilityType) {
+                                            case APP_SERVICES:
+                                                // App services only updates what was changed so we need
+                                                // to update the capability rather than override it
+                                                AppServicesCapabilities appServicesCapabilities = (AppServicesCapabilities) capability;
+                                                if (capability != null) {
+                                                    List<AppServiceCapability> appServicesCapabilitiesList = appServicesCapabilities.getAppServices();
+                                                    AppServicesCapabilities cachedAppServicesCapabilities = (AppServicesCapabilities) cachedSystemCapabilities.get(systemCapabilityType);
+                                                    //Update the cached app services
+                                                    if (cachedAppServicesCapabilities != null) {
+                                                    	cachedAppServicesCapabilities.updateAppServices(appServicesCapabilitiesList);
+                                                    }
+                                                    //Set the new capability object to the updated cached capabilities
+                                                    capability = cachedAppServicesCapabilities;
+                                                }
+                                                break;
+                                            case DISPLAYS:
+                                                shouldConvertDeprecatedDisplayCapabilities = false; // Successfully got DISPLAYS data. No conversion needed anymore
+                                                // this notification can return only affected windows (hence not all windows)
+                                                List<DisplayCapability> newCapabilities = (List<DisplayCapability>) capability;
+                                                updateCachedDisplayCapabilityList(newCapabilities);
+                                                systemCapabilitiesSubscriptionStatus.put(SystemCapabilityType.DISPLAYS, true);
+                                                break;
+											case VIDEO_STREAMING:
```

Comment:
On line 341, the alignment of `case VIDEO_STREAMING` needs to be fixed. It (and all of its code) needs to be on the same level as the case above it (`case DISPLAYS`).

Reply:
> On line 341, the alignment of `case VIDEO_STREAMING` needs to be fixed. It (and all of its code) needs to be on the same level as the case above it (`case DISPLAYS`).

@kostyaBoss I still see this issue.  On line 335, the alignment of `case VIDEO_STREAMING` needs to be fixed. It (and all of its code) needs to be on the same level as the case above it (`case DISPLAYS`).